app/assets/src/components/visualizations/heatmap/Heatmap.js

```diff
@@ -548,7 +584,10 @@ export default class Heatmap {
       this.options.marginTop,
       Math.max(deltaY + gCurrentTranslate[1], yScrollMax)
     );
-    this.g.attr("transform", `translate(${[dx, dy]})`);
+    this.g
+      .transition()
+      .duration(transition ? 750 : 0)
```

Comment:
nit: Move 750 to option above.

Reply:
I realized there's an existing `transitionDuration` option, so I'll use that instead.